data/brands/shop/department_store.json

```diff
@@ -12,6 +12,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "Reliance Smart Bazar",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Mall","Jio Mall"],
+      "tags": {
+        "brand": "Reliance Smart",
+        "brand:wikidata": "Q7311028",
+        "name": "Reliance Smart Bazar",
+        "shop": "department_store"
+      }
+    },
```

Comment:
Looks like this is a hypermarket (combination supermarket and department store). As far as I can tell, Reliance aquired most of the previous Big Bazaar stores (but not all, and the deal to buy Big Bazaar's owner outright fell through) and rebranded them to Smart Bazaar. Big Bazaar is [in NSI](https://nsi.guide/index.html?t=brands&k=shop&v=supermarket&tt=big%20bazaar#bigbazaar-9ef8f9) as a supermarket, and it's my understanding that that's the preferred tagging for hypermarkets absent a more specific tag being created. 

https://www.wikidata.org/wiki/Q3631008 is the WD item for the former Big Bazaar. It (and the associated enwiki article) have been changed to Smart Bazaar, but I'm not sure if that's entirely correct.

tl;dr:

- Better as `shop=supermarket`
- Might replace Big Bazaar, if that brand has fully ceased operations (I can't quite tell).
- Unsure whether to use Q3631008 or a new WD item.

Reply:
As per my local knowledge:

- **Smart Point** are small convenience stores (at least 10-20 locations dotted around in an avg Indian city)

- **Smart Bazaar** (rebranded from Big Bazaar) are usually multi story buildings (best suits `department_store` in osm wiki)

- **Smart Super Store** are Walmart equivalent  one large continuous store under the same roof